symfony/templates/participantlookup/participants-list.html.twig

```diff
@@ -30,23 +30,23 @@
             <tbody>
                 {% for participant in participants %}
                     <tr>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.lastName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.firstName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.middleName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.dob ? participant.dob|date('n/j/Y') }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.zipCode }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.lastName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.firstName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.middleName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.dob ? participant.dob|date('n/j/Y') }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.zipCode }}</a></td>
                         <td>
                             {% if participant.hpoId %}
-                                {% if participant.hpoId != app.siteOrganization %}
+                                {% if participant.hpoId != siteInfo.getSiteOrganization %}
```

Comment:
In silex `siteOrganization` return site's awardee. In Symfony I changed it so we need to use `getSiteAwardee` to get the site's awardee. Also, we can change `participant.hpoId` to `pariticipant.awardee`

```suggestion
                                {% if participant.awardee != siteInfo.getSiteAwardee %}
```